src/shared_realm.cpp

```diff
@@ -666,16 +717,19 @@ uint64_t Realm::get_schema_version(const Realm::Config &config)
 
 void Realm::close()
 {
+    // Order matters here: if we're the last Realm instance for this path, we
+    // need to tear down the coordinator before destroying our shared group
+    // to maintain the coordinator's invariant that there be a continuous session
```

Comment:
I'm not clear which invariant this is maintaining. What do you mean by "continuous session"?